test/media_sources/twitter_media_source_test.rb

```diff
@@ -43,6 +43,15 @@ def test_initializing_returns_blank
     tweets.each { |tweet| assert_not_nil(tweet.aws_image_keys) }
   end
 
+  test "tweet has screenshot aws key" do
+    skip unless ENV["AWS_REGION"].present?
+
+    tweets = TwitterMediaSource.extract(Scrape.create({ url: "https://twitter.com/Space4Europe/status/1552221138037755904" }))
+    assert_not_nil(tweets.first.aws_screenshot_key)
+
+    tweets.each { |tweet| assert_not_nil(tweet.aws_image_keys) }
```

Comment:
I believe this should be `tweet.aws_screenshot_key`, right?

Reply:
No, `tweets` is an array so I'm taking the first element.

Reply:
Er. nvm.